### PR TITLE
Electron draft: Sharing and Distributing apps

### PR DESCRIPTION
**Pull request by brianamarie**
## Overview
**TL;DR**
This pull request is for the draft of the third course in the [Electron path](https://github.com/github/training-kit/projects/2), Sharing Apps.

### Questions
- Does this course cover the right amount of breadth and depth? 
- Is the native app info accurate?
- Can GitHub Pages work for hosting apps? 
- Would GitHub releases work for hosting apps?
- If not GitHub pages, what type of website is needed?
- What other hosting is recommended?

### Next Steps
- ✏️ @github/services-training to fill in content 
- 💻 @github/services-training to test the activity
- 👍 and 💭 from @lee-dohm if we are moving in the right direction 

### Review
Not yet ready for review.

### CODE REVIEW

**Comment by hectorsector** [comment]
@brianamarie I'm happy to help ✋ fill this content in, but I'm curious whether we're leaping too far from "build a started app" to "get your app published". What do you think?

**Comment by brianamarie** [comment]
> I'm curious whether we're leaping too far from "build a starter app" to "get your app published". What do you think?

@hectorsector The purpose of this unit isn't so much to getting it published, but to get it shared in any way. We will share resources for publishing on app stores, but the tutorial portion will be  about uploading the application so others can download it from a personal page. Do you think that's out of scope?

**Comment by hectorsector** [comment]
> Do you think that's out of scope?

Nope, that sounds appropriate. Thanks for the clarification!

**Comment by brianamarie** [comment]
Merging drafts so we can start a clean, more formal review. 🚢
